jax/experimental/README.md

```diff
@@ -5,6 +5,10 @@ libraries are in part about providing tools and in part about serving as
 examples for how to build such libraries using JAX. Each one is only a few
 hundred lines of code, so take a look inside and adapt them as you need!
 
+#### 👉 **Note**: These mini-libraries are meant to be an _inspiration_, but not a prescription. 
+
+To serve that purpose, it is best to keep their code samples minimal; so we generally *will not merge PRs* adding new features. Instead, please send your lovely pull requests and design ideas to more fully-featured libraries like [DeepMind's Haiku](https://github.com/deepmind/dm-haiku) and Google Research's [FLAX](https://github.com/google/flax) or [Trax](https://github.com/google/trax)."
```

Comment:
Quick fix: extra `"` at the end of the paragraph

Reply:
@danielsuo: Am not sure why we would need a trailing `"` at the end of the paragraph - am I missing an opening quote? :)

Reply:
Haha, I just meant there's an extra `"` that renders at the end of the paragraph i.e., tiny typo (image below). Great contribution from yours truly, I know!

![image](https://user-images.githubusercontent.com/2692566/80482545-64210380-8922-11ea-8c74-cd6691ef66f8.png)

Reply:
Ah, gotcha! Thanks, @danielsuo.